src/main/java/com/exasol/releaserobot/usecases/validate/PlatformValidator.java

```diff
@@ -9,7 +10,9 @@
     /**
      * Validate a project.
      *
+     * @param repository instance of {@link Repository}
+     *
      * @return validation report
      */
-    public Report validate();
+    public Report validate(final Repository repository);
```

Comment:
I think this interface is not needed anymore, as we have the RepositoryValidator which does pretty much the same. Something to think about in another PR